src/components/NewMeasure/CreateNewMeasureDialog.tsx

```diff
@@ -369,22 +369,22 @@ const CreateNewMeasureDialog = ({ open, onClose }) => {
                 onChange={(startDate) => {
                   formik.setFieldValue("measurementPeriodStart", startDate);
                 }}
-                renderInput={(params) => (
-                  <TextField
-                    {...params}
-                    required
-                    helperText={formikErrorHandler(
-                      "measurementPeriodStart",
-                      true
-                    )}
-                    error={
-                      formik.touched.measurementPeriodStart &&
-                      Boolean(formik.errors.measurementPeriodStart)
-                    }
-                    {...formik.getFieldProps("measurementPeriodStart")}
-                    data-testid="measurement-period-start"
-                  />
```

Comment:
Do we have another PR to fix the same issue in madie-measure while a measure is being edited?

Reply:
Yup we will have a PR for that I am working on it